DotNet/overrides.js

```diff
@@ -65,4 +65,6 @@
 	'adopted styleSheets on firefox': {skipEmit: true},
 	'check region by selector within shadow dom with vg': {skipEmit: true},
 	'check region by element within shadow dom with vg': {skipEmit: true},
+	'pageCoverage data is correct': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
	"pageCoverage data is correct": {skipEmit: true},
```